pkg/utils/kubeconfig/kubeconfig.go

```diff
@@ -26,8 +29,16 @@ const (
 	AWSEKSAuthenticator = "aws"
 	// Shadowing the default kubeconfig path environment variable
 	RecommendedConfigPathEnvVar = clientcmd.RecommendedConfigPathEnvVar
+	// AWSIAMAuthenticatorMinimumBetaVersion this is the minimum version at which aws-iam-authenticator uses v1beta1 as APIVersion
+	AWSIAMAuthenticatorMinimumBetaVersion = "0.5.3"
+
+	alphaAPIVersion = "client.authentication.k8s.io/v1alpha1"
+	betaAPIVersion  = "client.authentication.k8s.io/v1beta1"
 )
 
+// CheckAuthenticatorVersion contains the function which is responsible to detect aws-iam-authenticator version.
+var CheckAuthenticatorVersion = getAWSIAMAuthenticatorVersion
```

Comment:
```suggestion
var checkAuthenticatorVersion = getAWSIAMAuthenticatorVersion
```

can we have this private and use the `export_test.go` pattern instead to set this?

Reply:
> no, because export_test.go is in the same package and not in `_test`. I don't know the reason for that though.

that sounds terrible.. can we just change it to be part of the test package?

Reply:
Ah I see what you mean. :D Okay.

Reply:
Probably, yeah. That's what I'm gonna do.